docs/walkthrough.md

```diff
@@ -0,0 +1,500 @@
+Walkthrough
+===========
+
+This walkthrough will go over the basics of setting up the Prometheus
+adapter on your cluster and configuring an autoscaler to use application
+metrics sourced from the adapter.
+
+Prerequisites
+-------------
+
+### Cluster Configuration ###
+
+Before getting started, ensure that the main components of your
+cluster are configured for autoscaling on custom metrics.  As of
+Kubernetes 1.7, this requires enabling the aggregation layer on the API
+server and configuring the controller manager to use the metrics APIs via
+their REST clients.
+
+Detailed instructions can be found in the Kubernetes documentation under
+[Horizontal Pod
+Autoscaling](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#support-for-custom-metrics).
+
+Make sure that you've properly configured Heapster with the `--api-server`
```

Comment:
that flag just say "run Heapster using the standard API server code as opposed to the home-grown go-restful code"